contxt/cli/clients.py

```diff
@@ -19,35 +23,41 @@ class Clients:
     """Holds a user and all client API's"""
 
     auth: CliAuth
+    load_dotenv()
+    env: str = os.getenv("env", "production")
```

Comment:
```suggestion
    env: str = os.getenv("CONTXT_ENV", "production")
```

since you're already doing this, can we just add a `env` keyword arg to all the classes in `contxt/auth/cli.py` that need to switch on env? that way, this line is the sole decider of the current env.

Reply:
good point, will do